configure.ac

```diff
@@ -3754,6 +3754,9 @@ CURL_CHECK_FUNC_STRERROR_R
 CURL_CHECK_FUNC_STRICMP
 CURL_CHECK_FUNC_STRTOK_R
 CURL_CHECK_FUNC_STRTOLL
+CURL_CHECK_FUNC_BZERO
+CURL_CHECK_FUNC_EXPLICIT_BZERO
+CURL_CHECK_FUNC_MEMSET_S
```

Comment:
Probably here too, with `"$curl_cv_native_windows" != "yes"`.

Reply:
Wrapped this in a windows check.